api/src/main/java/org/openmrs/api/UserService.java

```diff
@@ -65,7 +65,7 @@
 	 */
 	@Authorized( { PrivilegeConstants.EDIT_USER_PASSWORDS })
 	@Logging(ignoredArgumentIndexes = { 1, 2 })
-	public void changePassword(User user, String oldPassword, String newPassword) throws APIException;
+	public void changePassword(User user, String secret, String newPassword) throws APIException;
```

Comment:
@wluyima I just changed the variable name because it accepts either old password or answer to secret question, `secret` sounded like a better name than `oldPassword`.